### ISSUE TITLE
Add the latest version of ATBD doc

### ISSUE THREAD

**Issue by yunjunz** [nisar-solid/ATBD#36]
It would be helpful to keep the latest version of the solid Earth ATBD documents in PDF here in the repo, similar to what the cryo team has (https://github.com/fastice/nisarCryoCalValNotebooks/tree/main/ATBD-Documents).

**Comment by EJFielding** [comment]
Yes, it would be helpful if someone did that. Adrian does not upload to GitHub, so someone else would have to do it. I see the Cryo team also included a version number on their PDF. We have not been assigning any version numbers to our notebooks.

From: Zhang Yunjun ***@***.***>
Reply-To: nisar-solid/ATBD ***@***.***>
Date: Sunday, July 3, 2022 at 12:55 PM
To: nisar-solid/ATBD ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [nisar-solid/ATBD] Add the latest version of ATBD doc (Issue #36)


It would be helpful to keep the latest version of the solid Earth ATBD documents in PDF here in the repo, similar to what the cryo team has (https://github.com/fastice/nisarCryoCalValNotebooks/tree/main/ATBD-Documents<https://urldefense.us/v3/__https:/github.com/fastice/nisarCryoCalValNotebooks/tree/main/ATBD-Documents__;!!PvBDto6Hs4WbVuu7!Z6CGKk9ni16X6bSN3u8Rp-f_Nrp3KpXSJn-n7pwcqW3l7vkkM1aWrg-6RnjAb0G1PgVlVPFd5A$>).

**Comment by yunjunz** [comment]
For the PDF, it makes sense to have a version/revision number. For notebooks, we will have versions once we make a release of the repo; no extra versioning is needed inside each notebook.

**Comment by EJFielding** [comment]
I looked at that GitHub page, and it has only the main Cryo ATBD document in Word and PDF format, not a printout of their ATBD notebooks. I misunderstood what you were suggesting. I think Adrian has been putting version numbers on the main ATBD description document, but he has not uploaded them to GitHub.

Now we are looking at changing the way we calculate the Secular velocities in the next week or so, it might be premature to upload the present version of the ATBD document.

**Comment by yunjunz** [comment]
Then let's update the document after the changes.

**Comment by EJFielding** [comment]
We had another request from Paul Rosen to upload the ATBD description document, so I will try to get it from Adrian to upload.
